jbpm-case-mgmt/jbpm-case-mgmt-impl/src/main/java/org/jbpm/casemgmt/impl/CaseRuntimeDataServiceImpl.java

```diff
@@ -494,17 +507,19 @@ private int compareMilestones(CaseMilestoneInstance o1, CaseMilestoneInstance o2
         return processInstances;
     }
 
-
     @Override
     public Collection<CaseInstance> getCaseInstancesOwnedBy(String owner, List<CaseStatus> statuses, QueryContext queryContext) {
+        return getCaseInstancesOwnedBy(owner,statuses, false, queryContext);
```

Comment:
Could you add a test for coverage this method? I think it's the only one missing for complete coverage